lib/events.ts

```diff
@@ -48,14 +53,44 @@ export async function trackCommand(commandSignature: string) {
 	Sentry.configureScope(scope => {
 		scope.setExtra('command', commandSignature);
 	});
-	const balena = getBalenaSdk();
-	const balenaUrlPromise = balena.settings.get('balenaUrl');
+	const settings = await import('balena-settings-client');
+	const balenaUrl = settings.get('balenaUrl') as string;
+
 	return Bluebird.props({
-		balenaUrl: balenaUrlPromise,
-		username: balena.auth.whoami().catchReturn(undefined),
-		mixpanel: balenaUrlPromise.then(getMixpanel),
+		username: (async () => {
+			const getStorage = await import('balena-settings-storage');
+			const dataDirectory = settings.get('dataDirectory') as string;
+			const storage = getStorage({ dataDirectory });
+			let token;
+			try {
+				token = await storage.get('token');
```

Comment:
Not a blocker and just confirming that you had that in mind, I think that's pretty much the same as what `balena.auth.getToken()` does, but I'm not sure whether you have some extra module loading from the SDK that you want to avoid.